### PR TITLE
[INDY-2154] get rid of old request handlers

### PR DESCRIPTION
**Pull request by anikitinDSR**
Signed-off-by: Andrew Nikitin <andrew.nikitin@dsr-corporation.com>

### CODE REVIEW

File: indy_node/server/request_handlers/config_req_handlers/txn_author_agreement_aml_handler.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ class TxnAuthorAgreementAmlHandler(PTxnAuthorAgreementAmlHandler):
 
     def __init__(self, database_manager: DatabaseManager, bls_crypto_verifier,
```
Comment (skhoroshavin) [inline indy_node/server/request_handlers/config_req_handlers/txn_author_agreement_aml_handler.py]: Do we need `bls_crypto_verifier` here?

File: indy_node/server/request_handlers/config_req_handlers/txn_author_agreement_handler.py [hunk 2]
```diff
@@ -9,7 +9,7 @@ class TxnAuthorAgreementHandler(PTxnAuthorAgreementHandler):
 
     def __init__(self, database_manager: DatabaseManager, bls_crypto_verifier,
```
Comment (skhoroshavin) [inline indy_node/server/request_handlers/config_req_handlers/txn_author_agreement_handler.py]: Do we need `bls_crypto_verifier` here?

File: indy_common/test/auth/conftest.py [hunk 3]
```diff
@@ -52,71 +52,6 @@ def action_edit():
                           new_value='new_value')
 
 
-@pytest.fixture(scope='module')
-def idr_cache():
```
Comment (skhoroshavin) [inline indy_common/test/auth/conftest.py]: Did we move this fixture somewhere?

**Comment by anikitinDSR** [comment]
test this please

**Pull request merged**